packages/mds-agency-cache/index.ts

```diff
@@ -209,13 +209,13 @@ async function hwrite(suffix: string, item: CacheReadDeviceResult | Telemetry |
   if (nulls.length > 0) {
     // redis doesn't store null keys, so we have to delete them
     // TODO unit-test
-    await (await getClient()).hdelAsync(decorateKey(key), ...nulls)
+    await (await getClient()).hdelAsync(key, ...nulls)
```

Comment:
not clear to me why you removed `decorateKey`.  i get why you added it in the other places ...

Reply:
nvm i got it